less/global.less

```diff
@@ -16,22 +16,38 @@ h1, h2, h3, h4, h5 {
 
 h1 {
   font-size: 4rem;
+
```

Comment:
From looking at the design file, I think these media queries make the font too small. The very top "Fun Bus" does get smaller on mobile, but It hink the rest of the text is the same size as desktop 🤔